scaffolder-templates/skeletons/basic-workflow-skeleton/catalog-info.yaml

```diff
@@ -9,6 +9,7 @@ metadata:
     backstage.io/kubernetes-id: ${{ values.workflowId }}-ci
     janus-idp.io/tekton: ${{ values.workflowId }}
     github.com/project-slug: ${{ values.orgName }}/${{ values.repoName }}
+    gitlab.com/project-slug: ${{ values.orgName }}/${{ values.repoName }}
```

Comment:
this needs to be set for the advanced workflow as well I think

Reply:
it is there as well

Reply:
ah, right I missed it in the mass of the changes because the files moved, my bad